### ISSUE TITLE
SDXL checkpoints

### ISSUE THREAD

**Issue by scarbain** [HL-hanlin/Ctrl-Adapter#2]
Hello, thanks for the amazing paper and the release of i2vgenxl_depth !

When do you plan on releasing the models for SDXL ? Which one have you already trained ?

Also, do you have an idea on when you'll release the training code for SDXL ? :)

**Comment by HL-hanlin** [comment]
Thanks for your interest in our work! 

Yeah I'll working on cleaning the code for SDXL and other backbones in the next few days. Hopefully I can release the inference code and model checkpoint for SDXL this week, and I'll see if I can also finish cleaning the training code.

**Comment by 1585511010** [comment]
release this week？But I do not find it .

**Comment by HL-hanlin** [comment]
Working on it now. Just a few hours. Thanks!

**Comment by HL-hanlin** [comment]
SDXL inference code & checkpoint uploaded. Thanks for your patience!

**Comment by 1585511010** [comment]
could you please ask you when you release training code and training guideline to adapt our Ctrl-Adapter to new image/video diffusion models.

**Comment by HL-hanlin** [comment]
Hi all,
Thanks for your patience! The training code for SDXL has been uploaded. Please let me know if you find any issues!

**Comment by scarbain** [comment]
That's amazing, thanks !!

Can you please provide the non-default hyperparameters you used for training the depth SDXL CTRL-Adapter ?  Such as LR, warmup, precision, decay, ...

**Comment by HL-hanlin** [comment]
Hi, the hyper-parameters we used are all contained in the configuration files and training scripts. So no change needed! You can just run the scripts directly :)

**Comment by scarbain** [comment]
That's perfect, thanks Han!
